profile/README.md

```diff
@@ -0,0 +1,11 @@
+# Welcome to Pulsar
+
+Pulsar, or sometimes referred to as Pulsar-Edit is a new image of the beloved 'Hackable Text Editor' Atom.
+
+After the announcement of Atom's sunset the community came together to keep Atom alive, over on a longstanding fork [Atom-Community](https://github.com/atom-community/atom).
+
+Although due to the difference in goals for the editor long term a new version was born Pulsar.
+
+Pulsar aims to not only reach feature parity with origin Atom, but to bring Pulsar into the 21st century. With updates to the underlying dependencies and modern features supported.
```

Comment:
```suggestion
Pulsar aims to not only reach feature parity with the original Atom, but to bring Pulsar into the 21st century, updating the underlying architecture and supporting modern features.
```

Reply:
maybe something like this? I think underlying architecture would include the dependencies and is a little broader in scope

Reply:
Great idea, to allow ourselves some leeway from this initial claim.